src/domain/safe/entities/multisig-transaction.entity.ts

```diff
@@ -14,7 +14,7 @@ export type MultisigTransaction = {
   blockNumber: number | null;
   confirmations: Confirmation[] | null;
   confirmationsRequired: number;
-  data: string | null; // TODO will be added under https://github.com/5afe/safe-client-gateway-nest/pull/132
```

Comment:
Wasn't this comment about replacing `string | null` with the new entity `DataDecoded`?

Reply:
You're right! I changed the TODO for a more explicit message in [b20649a](https://github.com/safe-global/safe-client-gateway-nest/pull/350/commits/b20649a386e71b7bddaf347cadd861d8a85435f7)